tasks.py

```
@@ -1,71 +1,153 @@
+@task
+def compile(c, clean=False, config='Release', number_of_jobs=-1):
+    """
+    Compiles Reaktoro by running CMake and building with `ninja`.
+    Assumes that the environment is already configured using:
+        conda devenv
+        [source] activate reaktoro
+    """
+    build_dir, artifacts_dir = _get_and_prepare_build_and_artifacts_dir(
+        c,
+        clean=clean,
+        build_subdirectory="build",
+    )
 
-        # `PYTHON_INSTALL_PREFIX` is configured to `artifacts_dir / 'python'`
-        # so that it won't "pollute" the Python environment when in develop
-        # mode.
+    cmake_command = _get_cmake_command(
+        c,
+        build_dir=build_dir,
+        artifacts_dir=artifacts_dir,
+        cmake_generator="Ninja",
+        config=config,
+    )
+    build_command = strip_and_join(f"""
+        cmake
+            --build .
+            --target install
+            --config {config}
+            --
+                {f"-j {number_of_jobs}" if number_of_jobs >= 0 else ""}
+                {"-d keeprsp" if sys.platform.startswith("win") else ""}
+    """)
 
-        c.run(strip_and_join(f"""
-            cmake -G "{CMAKE_GENERATOR}"
-            -A "{CMAKE_ARCH}"
-            -DBUILD_ALL=ON
-            -DPYTHON_INSTALL_PREFIX="{(relative_artifacts_dir / 'python').as_posix()}"
-            -DCMAKE_BUILD_TYPE={CONFIG}
-            -DCMAKE_INCLUDE_PATH="{CMAKE_INCLUDE_PATH}"
-            -DBOOST_INCLUDE_DIR="{CMAKE_INCLUDE_PATH}"
-            -DCMAKE_INSTALL_PREFIX="{relative_artifacts_dir.as_posix()}"
-            -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON
-            "-DTHIRDPARTY_COMMON_ARGS=-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON"
-            "{str(relative_root_dir)}"
-        """))
+    commands = [cmake_command, build_command]
+
+    if sys.platform.startswith('win'):
+        commands.insert(0, r'"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" amd64')
```

I suggest to ensure that this path exists, and fail hard if it doesn't

---

- [ ] Ensure PATH exists